device/samples/NetStandardMultiplexSample/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using System.Threading;
+
+namespace multiplexingDotNetCore
+{
+    class Program
+    {
+        // String containing IotHub Hostname:
+        //"hostname.azure-devices.net"
+        private const string IoTHubHostName = "<iothubhostname>.azure-devices.net";
+
+        //Number of devices for multiplexing
+        private static int MUX_DEVICES = 4;
+        private static DeviceClient[] deviceClients = new DeviceClient[MUX_DEVICES];
```

Comment:
Why are all of these static? To promote better coding practices we should have our samples properly use classes representing a single temperature/humidity sensor and simply instantiate multiple of those in the `Main` function.